internal/website/content/howto/docstore/open-collection.md

```diff
@@ -0,0 +1,187 @@
+---
+title: "Open a Collection"
+date: 2019-06-08T15:14:26-04:00
+draft: true
+weight: 1
+---
+
+The first step in using Docstore is connecting to your
+document store provider. Every document store provider is a little different, but the Go CDK
+lets you interact with all of them using the [`*docstore.Collection`][] type. 
```

Comment:
Optional: It might be worthwhile to add a paragraph after this one explaining why the portable API needs the ID field mapping specified, since there's a fair amount of this guide that describes it.

Reply:
Done.